html/static/html/studio.html

```diff
@@ -0,0 +1,20 @@
+<div class="wrapper-comp-editor" id="editor-tab" data-editor="{self.editor}">
+  <div class="html-editor editor">
+    <div class="row">
+      <textarea id="html5-textarea">{self.sanitized_data}</textarea>
```

Comment:
Shouldn't this be the original, unsanitized text? I don't think the text entered by the author (including comments and potentially unsafe code) should always be kept and edited as-is, even though the LMS only uses the sanitized version.

Otherwise, for example, an author might spend an hour writing some JS code, save this block, then go back to edit it and see the code they worked on for an hour is gone.

Reply:
Cool, so in the previous version of the XBLock was passing `strip=True` to the Cleaner class, now I added `self.data` in the template, and allowed `strip=False` so that the added content is safely visible on the LMS.